File: 2-semestre/uri-online-judge/c/2146.c
#include <stdio.h>

int main() {
    int senha, novaSenha;
    
    while (scanf("%d", &senha) != EOF) {
        novaSenha = senha - 1;
        
        printf("%d\n", novaSenha);
    }
    
    return 0;
}

File: 2-semestre/uri-online-judge/java/2146.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int senha, novaSenha;
        
        Scanner entrada = new Scanner(System.in);
    
        while (entrada.hasNext()) {
            senha = entrada.nextInt();
            novaSenha = senha - 1;
            
            System.out.printf("%d\n", novaSenha);
        }
    }
}
